add.go

```diff
@@ -68,6 +74,18 @@ func (b *Builder) Add(destination string, extract bool, source ...string) error
 			logrus.Errorf("error unmounting container: %v", err2)
 		}
 	}()
+	// Set ownership of the destination.  If this information is
```

Comment:
It's a fairly long time before the user variable is used from this point and a number of other errors that could happen before then.  I'm just wondering if it would make sense to move this new block of code down to be run just before the for statement on line 121.

Reply:
That's right, I missed that!